sdk/hardwaresecuritymodules/Azure.ResourceManager.HardwareSecurityModules/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Other Changes
 
-## 1.0.0-beta.1 (2023-10-30)
+## 1.0.0-beta.1 (2023-12-30)
```

Comment:
`1.0.0-beta.1` has already been published: https://www.nuget.org/packages/Azure.ResourceManager.HardwareSecurityModules/1.0.0-beta.1

You need to publish `1.0.0-beta.2`. The entry is already added in the `CHANGELOG.md`. And don't forget to update the verison in `.csproj` file.

Reply:
@archerzz - Updated the readme file, thank you. The .csproj file is already using  <Version>1.0.0-beta.2</Version>, so I did not update this